javascript/Task4/task4.js

```diff
@@ -0,0 +1,28 @@
+let arr = [];
+let maxSeconds = null;
+
+let random = randomSecondCount()
+for(let i=1; i < random; i++) {
+    let seconds = randomSecondCount();
+    if(seconds > maxSeconds) {
+        maxSeconds = seconds;
+    }
+    arr.push(printFunction(i, seconds));
+}  
+
+function printFunction(index, seconds) {
+    return new Promise((resolve) => {
+        setTimeout(() => {
+            console.log(index +"-n function, " + seconds + " seconds"); 
+            resolve();  
+        }, seconds * 1000);
+    })
+}
+
+function randomSecondCount() {
```

Comment:
the same as above about a similar method: the method should receive the range (max and min values).